lib/model/QubitMenu.php

```diff
@@ -103,6 +104,11 @@ public function getPath($options = [])
             $path = $url;
         }
 
+        if (isset($options['removeIndex']) && true === $options['removeIndex']) {
+            $regex_for_index = '/\A(\/index.php)/';
```

Comment:
Not a big deal, but we could consider `qubit_dev.php` here too.

Reply:
E.g.: https://github.com/artefactual/atom/blob/qa/2.x/plugins/sfEacPlugin/lib/sfEacPlugin.class.php#L604

Reply:
Done!